From: NIIBE Yutaka <gniibe@fsij.org>
Date: Tue, 20 Sep 2016 15:41:36 +0900
Subject: doc: minor fix for @xref.

* doc/yat2m.c (proc_texi_cmd): Captalize "see" for xref.

(cherry picked from commit 187e2ad24fb92210587beb779a1cc746821a169c)
---
 doc/yat2m.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/doc/yat2m.c b/doc/yat2m.c
index a63260b..23fc6ba 100644
--- a/doc/yat2m.c
+++ b/doc/yat2m.c
@@ -730,7 +730,7 @@ proc_texi_cmd (FILE *fp, const char *command, const char *rest, size_t len,
     { "anchor",  7 },
     { "cartouche", 1 },
     { "ref",     0, "[", "]" },
-    { "xref",    0, "see: [", "]" },
+    { "xref",    0, "See: [", "]" },
     { "pxref",   0, "see: [", "]" },
     { "uref",    0, "(\\fB", "\\fR)" },
     { "footnote",0, " ([", "])" },
